ValheimPlus/ObjectDB.cs

```diff
@@ -0,0 +1,64 @@
+﻿using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace ValheimPlus
```

Comment:
Can we remove this file?

Reply:
Yeah, it's mostly testing injection of new items, which I now have working locally so will be part of a later PR